mw-webapp/src/component/table/columns.tsx

```diff
@@ -56,6 +74,9 @@ const getBoolean = (cellValue: CellContext<DayReport, boolean>) => {
   );
 };
 
+/**
+ * Render cell with date
```

Comment:
Unconsistent name and comment, check everywhere

Reply:
Changed